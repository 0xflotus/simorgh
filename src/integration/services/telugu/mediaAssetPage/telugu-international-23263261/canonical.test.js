/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service telugu
 * @pathname /telugu/international-23263261
 * @hasBulletedList true
 */

import runCanonicalTests from '../../../../pages/mediaAssetPage/canonicalTests';

describe('Canonical telugu mediaAssetPage', runCanonicalTests);
