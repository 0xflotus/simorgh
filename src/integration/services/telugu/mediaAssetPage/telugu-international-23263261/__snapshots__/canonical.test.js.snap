// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical telugu mediaAssetPage A11y I can see the skip to content link 1`] = `"కంటెంట్‌కు వెళ్లండి"`;

exports[`Canonical telugu mediaAssetPage Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=89&amp;p=international::telugu.international.media_asset.23263261.page&amp;x1=[urn:bbc:cps:4aeed9e6-5f64-344e-89c8-b17e31301b9f]&amp;x2=[responsive]&amp;x3=[news-telugu]&amp;x4=[te]&amp;x7=[article-media-asset]&amp;x8=[simorgh-nojs]&amp;x9=[Test+Telugu+MAP+-+BBC+News+తెలుగు]&amp;x11=[2020-01-29T13:05:05.000Z]&amp;x12=[2020-01-29T13:05:05.000Z]\\"/>"`;

exports[`Canonical telugu mediaAssetPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/telugu"`;

exports[`Canonical telugu mediaAssetPage Footer I can see the footer copyright and external linking text 1`] = `"© 2020  BBC. ఇతర వెబ్‌సైట్లలో సమాచారానికి బీబీసీ బాధ్యత వహించదు. ఇతర వెబ్‌సైట్లకు మా లింకింగ్ విధానం గురించి తెలుసుకోండి."`;

exports[`Canonical telugu mediaAssetPage Header I can see the branding 1`] = `"BBC News, తెలుగు"`;

exports[`Canonical telugu mediaAssetPage I can see the bulleted list item 1`] = `"AntsMore antsSo many ants"`;

exports[`Canonical telugu mediaAssetPage I can see the headline 1`] = `"Test Telugu MAP"`;

exports[`Canonical telugu mediaAssetPage I can see the timestamp 1`] = `"29 జనవరి 2020"`;

exports[`Canonical telugu mediaAssetPage Media player embed A11y I can read the media player title 1`] = `"మీడియా ప్లేయర్"`;

exports[`Canonical telugu mediaAssetPage Media player embed I can see the media player embed 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/cps/telugu/international-23263261/p01k6msp/te"`;

exports[`Canonical telugu mediaAssetPage SEO Canonical link 1`] = `"http://localhost:7080/telugu/international-23263261"`;

exports[`Canonical telugu mediaAssetPage SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical telugu mediaAssetPage SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical telugu mediaAssetPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical telugu mediaAssetPage SEO Heading level 1 1`] = `"Test Telugu MAP"`;

exports[`Canonical telugu mediaAssetPage SEO Lang attribute 1`] = `"te"`;

exports[`Canonical telugu mediaAssetPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/telugu.png",
      "width": 1024,
    },
    "name": "BBC News తెలుగు",
    "noBylinesPolicy": "https://www.bbc.com/telugu/institutional-50420343#authorexpertise",
  },
  "dateModified": "2020-01-29T13:05:05.000Z",
  "datePublished": "2020-01-29T13:05:05.000Z",
  "headline": "Test Telugu MAP",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/telugu.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/telugu/international-23263261",
    "@type": "WebPage",
    "name": "Test Telugu MAP",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/telugu.png",
      "width": 1024,
    },
    "name": "BBC News తెలుగు",
    "publishingPrinciples": "https://www.bbc.com/telugu/institutional-50420343",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/telugu.png",
  "url": "http://localhost:7080/telugu/international-23263261",
}
`;

exports[`Canonical telugu mediaAssetPage SEO OG description 1`] = `"Ants..."`;

exports[`Canonical telugu mediaAssetPage SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/telugu.png"`;

exports[`Canonical telugu mediaAssetPage SEO OG image alt 1`] = `"BBC News తెలుగు"`;

exports[`Canonical telugu mediaAssetPage SEO OG locale 1`] = `"te-IN"`;

exports[`Canonical telugu mediaAssetPage SEO OG site name 1`] = `"BBC News తెలుగు"`;

exports[`Canonical telugu mediaAssetPage SEO OG title 1`] = `"Test Telugu MAP - BBC News తెలుగు"`;

exports[`Canonical telugu mediaAssetPage SEO OG type 1`] = `"article"`;

exports[`Canonical telugu mediaAssetPage SEO OG url 1`] = `"http://localhost:7080/telugu/international-23263261"`;

exports[`Canonical telugu mediaAssetPage SEO Page title 1`] = `"Test Telugu MAP - BBC News తెలుగు"`;

exports[`Canonical telugu mediaAssetPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical telugu mediaAssetPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical telugu mediaAssetPage SEO Twitter creator 1`] = `"@bbcnewstelugu"`;

exports[`Canonical telugu mediaAssetPage SEO Twitter description 1`] = `"Ants..."`;

exports[`Canonical telugu mediaAssetPage SEO Twitter image alt 1`] = `"BBC News తెలుగు"`;

exports[`Canonical telugu mediaAssetPage SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/telugu.png"`;

exports[`Canonical telugu mediaAssetPage SEO Twitter site 1`] = `"@bbcnewstelugu"`;

exports[`Canonical telugu mediaAssetPage SEO Twitter title 1`] = `"Test Telugu MAP - BBC News తెలుగు"`;
